Guided Investigation - MDATP Webshell Alerts.ipynb

```diff
@@ -0,0 +1,700 @@
+{
```

Comment:
<p>q as variable name again.</p><p> </p><p>Also worth checking that cmd_data returns a DataFrame with data in rather than just letting it throw a TypeError</p><p> </p><p>If you wanted to move these larger queries out of the Notebook you could consider adding these to MSTICpy's queries. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Fixed. I will consider adding these to MSTICPy's queries in a future version to clean to code up.</p>